src/websys_bridge.rs

```diff
@@ -155,20 +155,16 @@ pub fn attach_children<Ms>(el_vdom: &mut El<Ms>) {
 pub fn attach_el_and_children<Ms>(el_vdom: &mut El<Ms>, parent: &web_sys::Node) {
     // No parent means we're operating on the top-level element; append it to the main div.
     // This is how we call this function externally, ie not through recursion.
-
     let el_ws = el_vdom
         .node_ws
         .as_ref()
         .expect("Missing websys el in attach_el_and_children");
 
     // Append the element
 
-    // todo: This can occur with raw html elements, but am unsur eof the cause.
-    match parent.append_child(el_ws) {
-        Ok(_) => {}
-        Err(_) => {
-            crate::log("Minor problem with html element (append)");
-        }
+    // todo: This error can occur with raw html elements, but am unsure of the cause.
```

Comment:
Could you create an issue with an example so that we don't forget it and I can try to debug it?

Reply:
I haven't encountered this in a while, and don't think it comes up anymore. I've replaced it locally with an expect call, and removed the todo. Will post here if it shows up.